scripts/ci/main.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+"""Testing NV12 image encoding."""
+
+from __future__ import annotations
+
+import argparse
+import os
+from typing import Any
+
+import cv2
+import numpy as np
+import rerun as rr
+import rerun.blueprint as rrb
+
+
+def bgra2y_u_v24(bgra: Any, full_range: bool) -> np.ndarray:
+    if full_range:
+        yvu = cv2.cvtColor(bgra, cv2.COLOR_BGR2YCrCb)
+        y, v, u = cv2.split(yvu)
+    else:
+        yuv = cv2.cvtColor(bgra, cv2.COLOR_BGR2YUV)
+        y, u, v = cv2.split(yuv)
+    y = np.array(y).flatten()
+    u = np.array(u).flatten()
+    v = np.array(v).flatten()
+    yuv24 = np.concatenate((y, u, v))
+    return yuv24.astype(np.uint8)
```

Comment:
is there no python library for this? :/

Reply:
dunno, surely but looking and comparing this would have taken me more time than just writing it out myself